docs/docs/topics/tasks.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Tasks
+---
+
+# Tasks
+
+The **Tasks system** in Infrahub is designed to manage and control various backend operations with robust error reporting, improved supervision, and enhanced logging capabilities.
+In Infrahub 1.1, key backend tasks began to be migrated to this framework, and subsequent releases have further refined its functionality.
+
+In Infrahub 1.0, we integrated Prefect as the task manager to orchestrate workflows. The task system is comprised of:
```

Comment:
Just some thoughts about ordering here you begin by mentioning what we did in 1.1 and then after that continue what we did in 1.0. I think it should be the other way around.